lib/Dist/Zilla/PluginBundle/Milla.pm

```diff
@@ -127,7 +127,85 @@ Dist::Zilla::PluginBundle::Milla - Dist::Zilla plugin defaults for Milla
 This is a Dist::Zilla plugin bundle that implements the opinionated build
 process of Milla. Roughly equivalent to:
 
-  # TBD
+  Varirables first:
+  $INSTALLER  = installer from Milla options or ModuleBuildTiny
+  $BUILD_FILE = ( $INSTALLER =~ /MakeMaker/ ) ? 'Makefile.PL' : 'Build.PL'
+  $RELEASE_TO = ( $ENV{FAKE_RELEASE} )        ? 'FakeRelease' : 'UploadToCPAN';
+
+  [NameFromDirectory]
```

Comment:
I think it would be helpful here to turn all the plugin names into hyperlinks, so the user can look up the full documentation for all options (plus new options may be added over time).  e.g.:

    L<C<NameFromDirectory>|Dist::Zilla::Plugin::NameFromDirectory>